content/docs/dvclive/api-reference/live/index.md

```diff
@@ -38,6 +38,9 @@ other metadata.
 - `dir` - Location of the directory to store
   [outputs](/doc/dvclive/get-started#outputs).
 
+- `params_path` - `{Live.params_path}`. Location of the parameters logged via
```

Comment:
Should it be `{Live.params_path}.yaml` to match `summary_path` (sorry this wasn't raised in the core PR)?

Reply:
Sorry, got confused here. Can't it just be `params.yaml` since we got rid of the custom path?

Reply:
In which case, does it even need to be a documented attribute/property?

Reply:
I'd go with `{Live.dir}/params.yaml`